packages/api-specification/src/window/window-api.md

```diff
@@ -26,15 +26,23 @@ Object to set features of the new window (non browser only).
 * `alwaysOnTop: boolean` - whether the window is always on top of other windows.
 * `backgroundColor: string` - set the backgroundColor of the window before the page loads.
 * `child: boolean` - make the new window a child of the current window.
+* `center: boolean` - position the new window in the center of the main screen. (only works on new windows with new ids)
 * `frame: boolean` - create the new window with a frame.
+* `hasShadow: boolean` - show shadows for frameless windows.
+* `height: Integer` - window's height in pixels.
```

Comment:
Small 'i'? Or all the others capitals? Don't mind which but think it will look better if its consistent :)